react-native/react/actions/update.js

```diff
@@ -0,0 +1,130 @@
+/* @flow */
+
+import * as Constants from '../constants/update'
+import engine from '../engine'
+import moment from 'moment'
+
+import {updateUi} from '../constants/types/keybase_v1'
+import type {Update} from '../constants/types/flow-types'
+import type {ShowUpdateAction, RegisterUpdateListenerAction, OnCancelAction, OnSkipAction, OnSnoozeAction, OnUpdateAction, SetAlwaysUpdateAction} from '../constants/update'
+import type {Dispatch} from '../constants/types/flux'
+
+let updateResponse: ?(result: any/* UpdatePromptRes */) => void
+export const SNOOZE_TIME_SECS = 60 * 60 * 24
```

Comment:
either: 
This shouldn't be exported
or 
This should be moved to constants/actions